### PR TITLE
YDSK: improvising_aerialist.txt + support

### PR DESCRIPTION
**Pull request by Northmoc**

### CODE REVIEW

File: forge-gui/res/cardsfolder/i/improvising_aerialist.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Name:Improvising Aerialist
+ManaCost:1 W
+Types:Creature Human Survivor
+PT:3/2
+T:Mode$ Phase | Phase$ Main | PhaseCount$ 2 | ValidPlayer$ You | PresentDefined$ Self | IsPresent$ Card.tapped | Execute$ TrigPerpetual | TriggerDescription$ Survival — At the beginning of your second main phase, if CARDNAME is tapped, CARDNAME and the top creature card in your library without flying perpetually gain flying.
+SVar:TrigPerpetual:DB$ Pump | PumpZone$ Battlefield,Library | Defined$ Self & ValidLibrary Creature.TopLibraryCreature | KW$ Flying | Duration$ Perpetual
```
Comment (squee1968) [inline forge-gui/res/cardsfolder/i/improvising_aerialist.txt:6]: Shouldn't the defined parameter be Defined$ Self & ValidLibrary Creature.TopLibraryCreature.withoutFlying?

Comment (Northmoc) [reply forge-gui/res/cardsfolder/i/improvising_aerialist.txt:6]: The syntax doesn't work that way but your overall point is correct!

**Review by paulsnoops — DISMISSED** [review]

File: forge-game/src/main/java/forge/game/card/CardProperty.java [hunk 2]
```diff
@@ -650,6 +650,10 @@ public static boolean cardHasProperty(Card card, String property, Player sourceC
             if (cards.isEmpty() || !card.equals(cards.get(0))) {
                 return false;
             }
+        } else if (property.startsWith("TopLibrary_")) {
+            CardCollection cards = new CardCollection(card.getOwner().getCardsIn(ZoneType.Library));
```
Comment (tool4ever) [inline forge-game/src/main/java/forge/game/card/CardProperty.java:654]: new CardCollection() shouldn't be needed?

Comment (tool4ever) [reply forge-game/src/main/java/forge/game/card/CardProperty.java:654]: Ideally all these properties for Top + Bottom would be cleaned up together (with only the Collections.reverse as difference when needed) 🤔
but that could be done later, feel free to resolve

**Review by tool4ever — APPROVED** [review]
should work ok like this

**Pull request merged**